controllers/upgrade_handlers.go

```diff
@@ -273,44 +273,58 @@ func (u *UpgHandler) PostPivot(ctx context.Context, ibu *lcav1alpha1.ImageBasedU
 	u.Log.Info("Starting health check for different components")
 	err := CheckHealth(u.Client, u.Log)
 	if err != nil {
-		utils.SetUpgradeStatusFailed(ibu, err.Error())
-		u.autoRollbackIfEnabled(ibu, fmt.Sprintf("Rollback due to health check failure: %s", err))
-		return doNotRequeue(), nil
+		utils.SetUpgradeStatusInProgress(ibu, fmt.Sprintf("System health checks in progress: %s", err.Error()))
+		return requeueWithShortInterval(), nil
```

Comment:
if the there's an err...it means health is in progress. Maybe a bool or typed string can be used to indicate in progress would be better?

Reply:
Yes, if any of the health checks fail for whatever reason, I'd consider that an overall health check failure, which would in turn mean the health checks are "in progress" and will retry once requeued.

Reply:
Yeah i guess in this case there's no possibility of random errors like "missing CRDs" and any get/list api call fails would be implicitly retried this way. Alright sounds good.